core/declaration.ts

```diff
@@ -1,19 +1,18 @@
-import { IColumnsDescriptor, mapToColumnProtoArray, Session } from "@dataform/core/session";
+import { mapToColumnProtoArray, Session } from "@dataform/core/session";
 import { dataform } from "@dataform/protos";
+import { IColumnsDescriptor, ICommonOutputConfig } from "@dataform/core/common";
 
-export interface DConfig {
-  description?: string;
-  columns?: IColumnsDescriptor;
-  database?: string;
-  schema?: string;
-}
+export interface IDeclarationConfig extends ICommonOutputConfig {}
```

Comment:
this `extends` means that `IDeclarationConfig` ends up with a `dependencies` field which it doesn't have (and which is confusing IMO).

Reply:
Woops, doesn't support tags either. Just refactored the interfaces a bit